panel/src/helpers/direction.js

```diff
@@ -1,29 +0,0 @@
-
-export default (app) => {
-  const defaultLanguage = app.$languages.default || null;
-  const language = app.$languages.current || null;
-  const multilang = app.$system.multilang || false;
-  const userLanguage = app.$user ? app.$user.language : null;
-  const direction = language ? language.direction : null;
-
-  /**
-   * Return LTR/RTL direction only when;
-   * - Multilang enabled
-   * - Current editing language exists
-   * - Input is not disabled
-   * - Editing language direction not equal with default language direction or
-   *   user language not equal with editing language
-   *
-   */
-  if (
-      multilang &&
-      language &&
-      app.disabled === false &&
```

Comment:
This line means: When field disabled like `translation: false`, direction should keep as default language. 

How can we handle right now?

Reply:
So it relied on that the component would have a `disabled` prop/computed prop?
We might have to do that in the components then:

```
:dir="direction"
```

to 

```
:dir="!disabled ? direction : null"
```

Reply:
We might even use a Vue directive to not repeat ourselves? https://v3.vuejs.org/guide/migration/custom-directives.html#_3-x-syntax

Reply:
Let me give you an example:

Default language: EN
Secondary language: AR (RTL)
Text field: `translate: false`

When you switch to secondary language, you'll see still EN content. So direction should be enabled only field editable.